apphub/image_generation/pggan_nihchestxray/pggan_nihchestxray.py

```diff
@@ -152,22 +151,28 @@ def on_epoch_end(self, state):
             for i in range(pred.shape[0]):
                 plt.subplot(4, 4, i + 1)
                 disp_img = pred[i].numpy()
-                disp_img -= disp_img.min()
-                disp_img /= (disp_img.max() + eps)
-                plt.imshow(disp_img)
-            plt.savefig(
-                os.path.join(self.save_dir,
-                             'image_at_{:08d}.png').format(state["epoch"]))
-            print("on epoch {}, saving image to {}".format(
-                state["epoch"], self.save_dir))
-
-
-def get_estimator():
-    train_csv, data_path = load_data()
+                if self.num_channels == 1:
```

Comment:
what happens when the `self.num_channels` is not 1?   I think there may be some indentation issue.

Reply:
Oh, you are right. Indentation seems to be messed up.. lemme fix this